Lib/concurrent/futures/_base.py

```diff
@@ -594,10 +599,21 @@ def map(self, fn, *iterables, timeout=None, chunksize=1):
                 before the given timeout.
             Exception: If fn(*args) raises for any values.
         """
+        if buffersize is not None and buffersize < 1:
```

Comment:
Why does it have to be None?

Reply:
I think you can just use `buffersize=0` to emulate the None part. Unless you plan to have 0 to be something different.

Reply:
@picnixz (thanks a lot for your time!)

I found missleading to make `buffersize=0` trigger the default (current) behavior, which is basically `buffersize=inf` 🤔. But I don't have a strong opinion on that point! what do you think?

Reply:
I don't have a strong preference. For instance, it's better to see what the rest of the code does (if there are similar cases). But keeping None is also fine actually.

Reply:
I'm ok with the None.